sematic/ui/src/components/ScrollingLogView.tsx

```diff
@@ -74,6 +69,21 @@ export default function ScrollingLogView(props: {
     return "Rendering...";
   }, [isAccumulating, isAccumulatorLoading, accumulatedLines, hasMore]);
 
+  const pullDownTriggerSection = useMemo(() => {
+    if (!pullDownTriggerEnabled || isAccumulating || !hasMore) {
+      return <></>;
+    }
+    return (<>
+      <span style={{marginLeft: "2em", fontSize: '14px'}}>Keep scrolling down to get more logs</span>
+      <LinearProgress value={Math.floor(pullDownProgress)} variant={"determinate"}
+      sx={{
+        '& .MuiLinearProgress-bar': {
+          'transitionDuration': '10ms'
+        }
+      }} />
```

Comment:
Can we use either a spinner or a progress bar, but not both?
This component already has a spinner to indicate loading, can we unify with this ?

Reply:
I second this. I personally prefer the spinner.

Reply:
Thanks for the suggestion. Per offline discussion, we decide to keep this experience, so the user can have some visual feedback to see how likely the pulldown function will trigger. Without the experience, the screen freezes when we reach the bottom of the scrolling area. And the user would get disoriented to know what's going on.

A comment is added to explain what the progress bar is for.